dragnet/model_training.py

```diff
@@ -199,7 +218,7 @@ def accuracy_auc(y, ypred, weights=None):
 
 
 def evaluate_models_tokens(datadir, dragnet_model, figname_root=None,
-    tokenizer=simple_tokenizer, cetr=False):
+    tokenizer=simple_tokenizer, cetr=False, content_or_comments='both'):
```

Comment:
I thought you only had two modes here, with or without comments. A Boolean would do for that.